tests/testConverterGeometries.py

```diff
@@ -0,0 +1,171 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 The Newton Developers
+# SPDX-License-Identifier: Apache-2.0
+import pathlib
+
+import usdex.core
+from pxr import Gf, Usd, UsdGeom
+
+from tests.util.ConverterTestCase import ConverterTestCase
+from urdf_usd_converter._impl.convert import Converter
+
+
+class TestConverterGeometries(ConverterTestCase):
+    def setUp(self):
+        super().setUp()
+        self.tolerance = 1e-6
+
+    def test_geometries(self):
+        input_path = "tests/data/simple-primitives.urdf"
+        output_dir = self.tmpDir()
+
+        converter = Converter()
+        asset_path = converter.convert(input_path, output_dir)
+        self.assertIsNotNone(asset_path)
+        self.assertTrue(pathlib.Path(asset_path.path).exists())
+
+        stage: Usd.Stage = Usd.Stage.Open(asset_path.path)
+        self.assertIsValidUsd(stage)
+
+        default_prim = stage.GetDefaultPrim()
+        geometry_scope_prim = stage.GetPrimAtPath(default_prim.GetPath().AppendChild("Geometry"))
+        self.assertIsNotNone(geometry_scope_prim)
+
+        link_box_prim = stage.GetPrimAtPath(geometry_scope_prim.GetPath().AppendChild("link_box"))
+        self.assertIsNotNone(link_box_prim)
+        self.assertTrue(link_box_prim.IsA(UsdGeom.Xform))
+
+        box_prim = stage.GetPrimAtPath(link_box_prim.GetPath().AppendChild("box"))
+        self.assertIsNotNone(box_prim)
+        self.assertTrue(box_prim.IsA(UsdGeom.Cube))
+        cube = UsdGeom.Cube(box_prim)
+        self.assertTrue(Gf.IsClose(usdex.core.getLocalTransform(box_prim.GetPrim()).GetTranslation(), (-0.5, 0.0, 0.5), self.tolerance))
```

Comment:
Why does this one use `self.tolerance` but the next line doesn't? I think just drop self.tolerance since it seems unlikely you'd want to control the global for all assertions